README.md

```diff
@@ -59,21 +53,20 @@ To install Procrustes using the conda package management system, install
 first, and then:
 
 ```bash
-    # Create and activate myenv conda environment (optional, but recommended)
-    conda create -n myenv python=3.6
-    conda activate myenv
+# Create and activate myenv conda environment (optional, but recommended)
+conda create -n myenv python=3.6
+conda activate myenv
 
-    # Install the stable release.
-    conda install -c theochem qc-procrustes
+# Install the stable release.
+conda install -c theochem qc-procrustes
```

Comment:
Thank you for the PR.

I don't think this part is necessary as the original makrdown documentation and GitHub instructions didn't impose such an indentation. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
Do you have a reason for putting 4 spaces in front of a shell command? If not, unnecessary white space is best avoided.